crates/swc/Cargo.toml

```diff
@@ -50,7 +50,7 @@ swc_common = {version = "0.17.0", path = "../swc_common", features = ["sourcemap
 swc_ecma_ast = {version = "0.65.3", path = "../swc_ecma_ast"}
 swc_ecma_codegen = {version = "0.89.1", path = "../swc_ecma_codegen"}
 swc_ecma_ext_transforms = {version = "0.50.0", path = "../swc_ecma_ext_transforms"}
-swc_ecma_lints = {version = "0.11.1", path = "../swc_ecma_lints"}
+swc_ecma_lints = {version = "0.11.1", path = "../swc_ecma_lints", features = ["non_critical_lints"]}
```

Comment:
This should not be enabled by the swc crate.
`node` and `wasm` should depend on `swc_ecma_lints` and enable this feature.

Reply:
Then tests in `swc` crate will fail

Reply:
Then we can add `swc_ecma_lints ` with features enabled to `[devDependencies]`, as we use 2021 edition.

Reply:
Sounds good